components/project/traces/trace-filter.tsx

```diff
@@ -0,0 +1,438 @@
+"use client";
+
+import { Button } from "@/components/ui/button";
+import { Checkbox } from "@/components/ui/checkbox";
+import {
+  Command,
+  CommandEmpty,
+  CommandGroup,
+  CommandInput,
+  CommandItem,
+} from "@/components/ui/command";
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+} from "@/components/ui/dialog"; // Adjust the import path as needed
+import { Input } from "@/components/ui/input";
+import {
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+} from "@/components/ui/popover";
+import {
+  ChromaDBMethods,
+  Event,
+  OpenAIMethods,
+  PineconeMethods,
+} from "@langtrase/trace-attributes";
```

Comment:
this approach wouldn't work anymore cos these attributes have moved over to the typescript SDK now. So we need to import it from the typescript SDK instead. Install the latest version of trace attributes and you will see this break.